src/Resources/contao/library/Contao/TemplateInheritance.php

```diff
@@ -78,7 +78,7 @@ public function inherit()
 		while ($this->strParent !== null)
 		{
 			$strCurrent = $this->strParent;
-			$strParent = $this->strDefault ?: \Controller::getTemplate($this->strParent, $this->strFormat);
+			$strParent = $this->strDefault ?: static::getTemplatePath($this->strParent, $this->strFormat);
```

Comment:
`$this->getTemplatePath()` would work. Should I change only the call to `$this->getTemplatePath()` or should I also make the method non-static?